FrameworkSystem/private/standardLogging/LoggingRoot.py

```diff
@@ -135,6 +128,67 @@ def initialize(self, systemName, cfgPath):
     finally:
       self._lockConfig.release()
 
+  def __getBackendsFromCFG(self, cfgPath):
+    """
+    Get backends from cfg file and register them in LoggingRoot. 
+    This is the new way to get the backends working with 'LogBackends' and 
+    'BackendsConfig' options.
+    We can use the entire power of the CS file to provide a general configuration
+    and many local configurations. 
+
+    :params cfgPath: string of the cfg file path
+    """
+    # We have to put the import line here to avoid a dependancy loop
+    from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+    from DIRAC import gConfig
+
+    # get the second last string representing the component name in the cfgPath
+    component = cfgPath.split("/")[-2]
+    operation = Operations()
+
+    # Search desired backends in the component
+    desiredBackends = gConfig.getValue("%s/%s" % (cfgPath, 'LogBackends'), [])
+    if not desiredBackends:
+      # Search desired backends in the operation section according to the component type
+      desiredBackends = operation.getValue("Logging/Default%sBackends" % component, [])
```

Comment:
This should not be here. In Operations we put only the default. If we want specific backend for a given component, we define it in the component itself